src/loaders/zarr_utils/types.ts

```diff
@@ -93,3 +93,20 @@ export type OMEZarrMetadata = {
 };
 
 export type NumericZarrArray = zarr.Array<zarr.NumberDataType, WrappedStore<RequestInit>>;
+
+/** A record with everything we need to access and use a single remote source of multiscale OME-Zarr data. */
+export type ZarrSource = {
+  /** Representations of each scale level in this zarr. We pick one and pass it to `zarrGet` to load data. */
+  scaleLevels: NumericZarrArray[];
+  /**
+   * Zarr dimensions may be ordered in many ways or missing altogether (e.g. TCXYZ, TYX). `axesTCZYX` represents
+   * dimension order as a mapping from dimensions to their indices in dimension-ordered arrays for this source.
+   */
+  axesTCZYX: TCZYX<number>;
+  /** OME-specified metadata record with most useful info on the current image, e.g. sizes, axis order, etc. */
+  multiscaleMetadata: OMEMultiscale;
+  /** OME-specified "transitional" metadata record which we mostly ignore, but which gives channel & volume names. */
+  omeroMetadata: OmeroTransitionalMetadata;
+  /** Which channels in the volume come out of this source - i.e. source channel 0 is volume channel `channelOffset` */
+  channelOffset: number;
+};
```

Comment:
This is the essential type added by this PR. Most of the fields correspond to fields removed from `OMEZarrLoader` because they describe just one zarr and we need to handle multiple. `channelOffset` is the only new one.